CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## [4.2.10]
+
+- update com.taoensso/nippy to "2.14.0" in order to address some clojure warnings
+- update com.taoensso/encore to "2.115.0" 
```

Comment:
Is there a reason these versions are in quotes when the rest are not?

Reply:
It was a copy/paste.  I can remove them.